au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml

```diff
@@ -34,7 +34,7 @@ StyledPopupView {
     FontMetrics {
         id: fontMetrics
 
-        font: content.font
+        font: ui.theme.defaultFont
```

Comment:
or should it be the font from StyledTextLabel?

Reply:
Of course it should, fixed